src/utils/notification-common.ts

```diff
@@ -0,0 +1,115 @@
+import { cva } from "class-variance-authority";
+
+/** The duration of the animation in the appearance and disappearance of the toast (in msec). */
+export const TOAST_ANIMATION_DURATION = 200;
+
+export function formatDate(date: Date): string {
+  return date.toLocaleDateString(undefined, { dateStyle: "medium" });
+}
+
+export const cvaContainer = cva(
+  [
+    "flex",
+    "items-center",
+    "gap-3",
+    "w-full",
```

Comment:
Add `h-full` to allow the user to change component height.

```suggestion
    "w-full",
    "h-full",
```